speculos/api/api.py

```diff
@@ -23,13 +23,13 @@
 class ApiRunner(IODevice):
     """Run the Speculos API server in a dedicated thread, with a notification when it stops"""
     def __init__(self, api_port: int) -> None:
-        self._app: Flask
+        self._app_wrapper: ApiWrapper
```

Comment:
Unused? And mismatch `_api_wrapper` below?

Reply:
Mismatch true